tests/test_app/src/Queue/Task/FooTask.php

```diff
@@ -30,7 +41,23 @@ class FooTask extends Task {
 	 * @return void
 	 */
 	public function run(array $data, int $jobId): void {
-		$this->out('CakePHP Foo Example.');
+		$this->io->out('CakePHP Foo Example.');
+		$test = $this->testService->output();
+		$this->io->out($test);
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function services(ContainerInterface $container): void {
```

Comment:
Why is this needed?
Wouldnt it be easier if the trait directly exposes the container method to get sth you need?